scripts.js

```diff
@@ -1 +1,4 @@
 console.log("Welcome, pilgrim.");
+function alertMe() {
+  alert ("I made a change!");
+}
```

Comment:
Handy little call-out by the Github UI. Looks like there should be a newline at the end of this file. I’ll go ahead and add it.